sample/build.gradle

```diff
@@ -4,41 +4,64 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'uk.gov.hmrc.spotless'
 apply plugin: 'io.gitlab.arturbosch.detekt'
 
+apply from: "./screenshots.gradle"
+
 detekt {
     config = files("detekt/config.yml")
     baseline = file("detekt/baseline.xml")
 }
 
 android {
+    compileSdkVersion compile_sdk_version
+    buildToolsVersion "30.0.3"
+
     viewBinding {
         enabled = true
     }
-    compileSdkVersion compile_sdk_version
+
     defaultConfig {
         applicationId "uk.gov.hmrc.components.sample"
         minSdkVersion min_sdk_version
         targetSdkVersion target_sdk_version
         versionCode 1
         versionName "1.0"
-        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+
+        testInstrumentationRunner "com.karumi.shot.ShotTestRunner"
+        testApplicationId "uk.gov.hmrc.components.sample.test"
     }
+
     buildTypes {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_11
         targetCompatibility JavaVersion.VERSION_11
     }
 
+    kotlinOptions {
+        jvmTarget = '11'
+    }
+
     lintOptions {
         lintConfig file('lint/config.xml')
         baseline file("lint/baseline.xml")
         warningsAsErrors true
         abortOnError true
     }
+
+    packagingOptions {
+        exclude 'META-INF/ASL2.0'
```

Comment:
Optional: We could do `exclude 'META-INF/*` to cover ourselves from other issues.

Reply:
Addressed in 694d064736478f073fa510eb663c87a79120f863